Source/Core/DolphinQt/ConvertDialog.cpp

```diff
@@ -329,6 +329,21 @@ void ConvertDialog::Convert()
     }
   }
 
+  if (std::any_of(m_files.begin(), m_files.end(), std::mem_fn(&UICommon::GameFile::IsNKit)))
+  {
+    if (!ShowAreYouSureDialog(
+            tr("Dolphin can't convert NKit files to non-NKit files. Converting an NKit file in "
+               "Dolphin will result in another NKit file.\n"
+               "\n"
+               "If you want to convert an NKit file to a non-NKit file, you can use the same "
+               "program as you originally used when converting the file to the NKit format.\n"
```

Comment:
I suggest the following wording: 

```suggestion
            tr("Dolphin cannot properly convert an NKit file to another file format."
               "Trying to, will convert it to a hybrid between NKit and the target format,"
               "that may not behave as expected.\n"
               "\n"
               "If you want to do a proper conversion, please use another program.\n"
```

Reply:
> Dolphin cannot properly convert an NKit file to another file format

It can properly convert an NKit file in one container format to an NKit file in another container format.

> Trying to, will convert it to a hybrid between NKit and the target format

Combining NKit with a compressed container format is not some kind of strange hybrid. .nkit.gcz is the default output of the NKit app.

> that may not behave as expected

I'd rather be clear about in what way it doesn't behave as expected, since it's simple to explain: The file will still be an NKit file. The problems you will have are the exact same as with any other NKit file.

Reply:
That's very complicated stuff, in which case I cannot think of a better wording than the one you are using. Still, it doesn't read particularly well.

Reply:
With that said, I'm willing to drop the second paragraph if there's more than one person who doesn't think we should include what I wrote there. (I would prefer dropping it over going with your variant, since your variant doesn't really provide any information that isn't already implied by the first paragraph.)